src/main.py

```diff
@@ -1,12 +1,16 @@
 import asyncio
+import random
+from os import remove
 from typing import Dict, Optional
 
 from nextcord import Attachment, Embed, Intents, Interaction, SlashOption, TextChannel
 from nextcord.ext import application_checks, commands
 
 import config
 import persistent_state
+import song_utils
 from bust import BustController, create_controller
+from discord_utils import filepath_builder, is_valid_media
```

Comment:
I would say for these also don't import the function, keep discord_utils when you call them. It can be confusing to find where they come from otherwise